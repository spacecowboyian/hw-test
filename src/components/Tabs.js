import React from 'react'
import { Row, Col } from 'react-flexbox-grid';
import { css } from 'emotion/macro';

import { spacing } from '../Styles';

const tabContainer = css`
  padding: 0 calc(${spacing}*8);
`

const tabs = css`
  list-style: none;
  margin: 0;
  padding: 0;
  display: flex;
`
const tab = css`
  padding: calc(${spacing}*8) 0;
  margin-right: calc(${spacing}*10);
  text-transform: uppercase;
  font-size: 15px;
  font-weight: 700;
  color: #003D40;
  border-bottom: ${spacing} solid transparent;
`
const tabActive = css`
  ${tab}
  color: #00E9CB; 
  border-bottom-color: #00E9CB; 
`

export default function Tabs() {
  return (
    <Row className={tabContainer}>
      <Col xs={12}>
        <ul className={tabs}>
          <li className={tabActive}>snapshot</li>
          <li className={tab}>trends</li>
        </ul>
      </Col>
    </Row>
  )
}
