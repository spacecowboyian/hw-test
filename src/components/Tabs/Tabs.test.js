import React from 'react';
import { cleanup } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import renderer from 'react-test-renderer';

import Tabs from './Tabs';

afterEach(cleanup);


it('renders correctly', () => {
  const tree = renderer
    .create(<Tabs />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
