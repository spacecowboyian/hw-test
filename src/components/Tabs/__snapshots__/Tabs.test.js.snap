// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="css-pxyi0o-tabContainer row"
>
  <div
    className="col-xs-12"
  >
    <ul
      className="css-1yca8z4-tabs"
    >
      <li
        className="css-mzdxzq-tab-tabActive"
      >
        snapshot
      </li>
      <li
        className="css-1qlp9uz-tab"
      >
        trends
      </li>
    </ul>
  </div>
</div>
`;
